src/app/_components/footer.tsx

```diff
@@ -0,0 +1,46 @@
+import Link from "next/link";
+import { Discord, Gmail, Instagram } from "./logo";
+
+export default function Footer() {
+    const mathAndCS = ["AP Calculus AB", "AP Calculus BC", "AP Statistics", "AP Computer Science Principles", "AP Computer Science A"]
+    const sciences = ["AP Biology", "AP Chemistry", "AP Environmental Science", "AP Physics 1", "AP Physics 2", "AP Physics C: E&M", "AP Physics C: M"]
+    const socialSciences = ["AP Psychology", "AP European History", "AP United States History", "AP World History", "AP Microeconomics", "AP Macroeconomics", "AP Human Geography", "AP Comparative Government and Politics", "AP US Government and Politics"]
+    const language = ["AP English Language", "AP English Literature", "AP Spanish Language", "AP Spanish Literature", "AP Chinese", "AP French", "AP German", "AP Italian", "AP Japanese", "AP Latin"]
+    const arts =  ["AP 2D Art and Design", "AP 3D Art and Design", "AP Drawing", "AP Art History", "AP Biology", "AP Music Theory", "AP Research", "AP Seminar" ]
+    return (
+        <div className="w-screen bg-gray-900 text-center">
```

Comment:
this structure is strange. you have a `w-screen` class, which is 100vw. the viewport width adds in the width of the scrollbar, so by using 100% of the viewport width, you're actually exceeding the space the browser gives you. `w-full` is 100% of the available space

though this could be done in flexbox, this could also be a grid with a defined number of columns (`grid-template-columns`). in Tailwind, the property for that is `grid-cols-x`. I mention CSS grids because you can define how columns are displayed. on smaller screens, for instance, the list of classes all appears as one continuous list. in a CSS grid, you can divide the columns into two and set one column, like the project information column, to have twice its span. this is harder in flexbox

Reply:
Thanks for the detailed explanation!